src/audio/up_down_mixer/coef/up_down_mixer.h

```diff
@@ -5,7 +5,7 @@
 // Author: Bartosz Kokoszko <bartoszx.kokoszko@intel.com>
 // Author: Adrian Bonislawski <adrian.bonislawski@intel.com>
 
-#include <ipc4/up_down_mixer.h>
+#include "../up_down_mixer_ipc4.h"
```

Comment:
can we just name this file "coef.h" and place it at the same level?

Reply:
this is not coef header, coef header put it to coef/up_down_mixer.h
this header have definition:
ipc4_up_down_mixer_module_cfg

Reply:
yes, I mean the coefficient header, i.e. the file, which is being patched here, not the one, included by this instruction, sorry for the confusion

Reply:
Do you mean change coef/up_down_mixer.h --> up_down_mixer_coef.h ?

Reply:
it is not coef.h, it is in coef folder with name up_down_mixer.h. 
downmix only have one header file, however, for src/asrc, we have hundred of header files, change file name is huge cost.
how about keep it with coef/up_down_mixer.h?

Reply:
> it is not coef.h, it is in coef folder with name up_down_mixer.h. downmix only have one header file, however, for src/asrc, we have hundred of header files, change file name is huge cost. how about keep it with coef/up_down_mixer.h?

@btian1 I know that your current version calls it "coef/up_down_mixer.h" and that's what I'm proposing to change for two reasons: (1) two files with the same name in adjacent directories look confusing, (2) no need to create an additional directory level just for one file. So, originally I proposed to just do
```
git mv coef/up_down_mixer.h coef.h
```
to which you proposed to call it up_down_mixer_coef.h instead which is also fine with me, but not using the same name as in your current version.

Reply:
ok, use up_down_mixer_coef.h is better since coef.h is too common for file name.